front/package.json

```diff
@@ -46,4 +48,4 @@
     "karma-jasmine-html-reporter": "~2.1.0",
     "typescript": "~5.2.2"
   }
-}
+}
```

Comment:
There's something wrong in your vsCode or you're manually removing empty lines in the end of the files (same for `app.component.html`) - please add empty lines back - it's considered good to have them and even github highlights their absence